api.py

```diff
@@ -410,6 +411,98 @@ def get_latest():
     return make_response(jsonify(result), 200)
 
 
+@app.route('/all', methods=['GET'])
+def kml_maker_all():
+    try:
+        # Attempt to make a connection to the Database
+        conn = psycopg2.connect(
+             "dbname=" + Secrets.dbname +
+             " user=" + Secrets.username +
+             " host=" + Secrets.hostname +
+             " password=" + Secrets.password)
+    except:
+        # On Failure, output a message saying so and retreat
+        return ""
+
+
+    # Create a psycopg2 cursor for the DB
+    curr = conn.cursor()
+
+    query = "SELECT rideid FROM rides"  
+    curr.execute(query)
+
+    all_routes = []
+    
+    for ride in curr.fetchall():
+	pointQ = "SELECT ST_X(point) AS long, ST_Y(point) AS LAT FROM points WHERE rideid = %s AND accuracy <= 20"
+	statsQ = "SELECT * from stats_view WHERE rideid = %s"
```

Comment:
`SELECT *` can lead to problems if the schema ever changes, is not explicit in what data is being requested, and often fetches more data than is needed.

Additionally, I think it may be possible to combine these two queries:

```SQL
SELECT
  ST_X(point) AS long,
  ST_Y(point) AS LAT,
  stats_view.*
FROM points
  INNER JOIN stats_view
    ON points.rideid = stats_view.rideid
WHERE points.rideid = %s
  AND points.accuracy <= 20
```